README.md

```diff
@@ -243,15 +190,28 @@ Code Quality (does not affect risk score):
 ┌───┬────────────────────────────────────────────────────────────────────────────────────────────┐
 │ ! │ This package version's size on disk is 40.0 kB.                                            │
 └───┴────────────────────────────────────────────────────────────────────────────────────────────┘
+
+Required By (leftmost is directly in your package):
+┌────────────────────────────────────────────────────────────────────────────────────────────┐
+│ (Directly in your package)                                                                 │
+└────────────────────────────────────────────────────────────────────────────────────────────┘
 ```
 
+## `ncm install <module{@version}>`
+
+Runs and displays `ncm details <module{@version}>` with an interactive confirmation prompt.
+If confirmed, attempts to run `npm install <module{@version}>` with any additional options provided.
+
+_The config keys `installBin` and `installCmd` can adjust this to work with other package installers if necessary._
```

Comment:
Maybe provide an example for this?

Reply:
Is see `ncm config --help` good enough? I dunno if it's worth doing more for than that...

Reply:
I think so yeah